controllers/home.js

```diff
@@ -15,10 +17,24 @@ async function home(req, res) {
   let featuredBreach = null;
   let scanFeaturedBreach = false;
 
+  // Growth Experiment
+  let experimentBranch = null;
+  let isUserInExperiment = null;
+  let experimentBranchB = null;
```

Comment:
`let` = code sniff. Can we wrap these up in a single object and do something more like:
```js
const experimentFlags = _getExperimentFlags(req);
...
res.render("dashboards", {
...
  experimentFlags,
});
```